### PR TITLE
chore(contentblockmedia): update story

### PR DESCRIPTION
**Pull request by andysherman2121**
### Related Ticket(s)

https://github.com/carbon-design-system/ibm-dotcom-library/issues/2969

### Description

remove menu json knobs and add prop names

### Changelog

**New**

prop names to labels

**Removed**

json knobs

### CODE REVIEW

File: packages/react/src/components/ContentBlockMedia/__stories__/ContentBlockMedia.stories.js [hunk 1]
```diff
@@ -158,26 +158,22 @@ WithAsideElements.story = {
 
         const linkListProps = {
           heading: text('link list heading:', 'Tutorials', groupId),
```
Comment (diegomarcuz) [inline packages/react/src/components/ContentBlockMedia/__stories__/ContentBlockMedia.stories.js:160]: Hey @andysherman2121, I know you didn't touch here for this cleanup, but just a suggestion.... if you change the `link list heading ` to `Link List (heading)`, what do you think?

Comment (andysherman2121) [reply packages/react/src/components/ContentBlockMedia/__stories__/ContentBlockMedia.stories.js:160]: Good idea @Diego-Marcuz. I'll add that now

File: packages/react/src/components/ContentBlockMedia/__stories__/ContentBlockMedia.stories.js [hunk 2]
```diff
@@ -157,7 +157,7 @@ WithAsideElements.story = {
         const items = [item, item];
 
         const linkListProps = {
-          heading: text('link list heading:', 'Tutorials', groupId),
+          heading: text('link list (heading)', 'Tutorials', groupId),
```
Comment (diegomarcuz) [inline packages/react/src/components/ContentBlockMedia/__stories__/ContentBlockMedia.stories.js:160]: @andysherman2121 thanks to add it. Just another minor comment... For the `link list`, is CamelCase good ? Like this: `Link List`.

Comment (andysherman2121) [reply packages/react/src/components/ContentBlockMedia/__stories__/ContentBlockMedia.stories.js:160]: @diegomarcuz hmm yeah maybe it should be `Link List (heading)`

Comment (diegomarcuz) [reply packages/react/src/components/ContentBlockMedia/__stories__/ContentBlockMedia.stories.js:160]: Awesome, thanks @andysherman2121

**Comment by asudoh** [comment]
Hi @andysherman2121 would you update the snapshot...? Thanks!

**Comment by andysherman2121** [comment]
> Hi @andysherman2121 would you update the snapshot...? Thanks!

@asudoh just updated! was running into some ci-check issues. Solved now

**Pull request merged**